api/v1/helpers/verbs/utils.js

```diff
@@ -841,6 +841,96 @@ function responsify(rec, props, method) {
   return o;
 } // responsify
 
+const collectorMap = {};
```

Comment:
collector map feels like it belongs in its own module (rather than adding code to a thousand-line module)

Reply:
Makes sense. I moved it to verbs/heartbeatUtils.js.